src/app/loginComponents/accounts/external/accounts.component.html

```diff
@@ -76,6 +76,10 @@ <h3>
       <span *ngIf="row.isLinked">
         <strong>Username: </strong>
         {{ row.source | getTokenUsername: tokens }}
+        <span *ngIf="row.name === 'ORCID'">
+          <img src="../../assets/images/account-logos/{{ row.logo }}" alt="ORCID iD logo" class="orcid-id-logo" />
+          <a href="https://orcid.org/{{ user.orcid }}" style="font-size: 0.8em"> {{ user.orcid }}</a>
```

Comment:
avoid inline style. this is to prevent our components from having a variety of different font-sizes